src/BaseGL.js

```diff
@@ -55,8 +63,20 @@ class BaseGL {
     // add events
     var self = this;
     this.plot.addEventListener("onSelectionEnd", (e) => {
+      e.preventDefault();
+      const sdata = e.detail.data;
+      if (
+        this.highlightEnabled &&
+        sdata &&
+        sdata.selection.indices.length > 0
+      ) {
+        this.highlightIndices(sdata.selection.indices, true);
+      }
+
       self.selectionCallback(e.detail.data);
```

Comment:
should this be more of an `if...else`? otherwise its going to call both highlight and selection callbacks?